components/certificate-mgt/org.wso2.carbon.identity.certificate.management/src/main/java/org/wso2/carbon/identity/certificate/management/model/Certificate.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.certificate.management.model;
+
+/**
+ * This class holds the certificate information.
+ */
+public class Certificate {
+
+    private final String id;
+    private final String name;
+    // Certificates only in PEM format are allowed.
+    private final String certificate;
+
+    public Certificate(Builder certificateBuilder) {
```

Comment:
With the builder better make this constructor private. So only builder should be used to create an instance. 
Even this make sure of that, but it's unnecessary api IMO

Reply:
Fixed with https://github.com/wso2/carbon-identity-framework/pull/6043/commits/524120310dc908182809894fb03869180f6c4ab8